agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/PhenotypeRepository.java

```diff
@@ -213,12 +213,12 @@ public long getDistinctPhenotypeCount(String geneID) {
     }
 
     public List<PhenotypeEntityJoin> getAllPhenotypeAnnotations() {
-        String cypher = "MATCH p0=(phenotype:Phenotype)--(phenotypeEntityJoin:PhenotypeEntityJoin)<-[:EVIDENCE]-(publications:Publication), " +
-                "p2=(phenotypeEntityJoin:PhenotypeEntityJoin)--(gene:Gene)-[:FROM_SPECIES]-(species:Species) " +
+        String cypher = "MATCH p0=(phenotype:Phenotype)--(pej:PhenotypeEntityJoin)-[:EVIDENCE]->(ppj:PhenotypePublicationJoin)<-[:ASSOCIATION]-(publication:Publication)," +
+                " p2=(pej:PhenotypeEntityJoin)<-[:ASSOCIATION]-(gene:Gene)-[:FROM_SPECIES]->(species:Species) " +
                 //"where gene.primaryKey = 'MGI:109583' " +
-                "OPTIONAL MATCH p4=(phenotypeEntityJoin:PhenotypeEntityJoin)--(feature:Feature)-[:CROSS_REFERENCE]->(crossRef:CrossReference) " +
-                "OPTIONAL MATCH models=(phenotypeEntityJoin:PhenotypeEntityJoin)--(:AffectedGenomicModel) " +
-                "return p0, p2, p4, models ";
+                "OPTIONAL MATCH     p4=(pej:PhenotypeEntityJoin)--(feature:Feature)-[:CROSS_REFERENCE]->(crossRef:CrossReference) " +
+                "OPTIONAL MATCH models=(ppj:PhenotypePublicationJoin)-[:PRIMARY_GENETIC_ENTITY]->(:AffectedGenomicModel) " +
```

Comment:
@cmpich one thing I have learned is that when using a `path` you don't always need to assign a variable to the node... unless you are going to use it later in the query. So a `(phenotypeEntityJoin:PhenotypeEntityJoin)--(gene:Gene)-[:FROM_SPECIES]-(species:Species)` could be changed to: `(:PhenotypeEntityJoin)--(:Gene)-[:FROM_SPECIES]-(:Species)`